c_src/ErlUtil.cc

```diff
@@ -1175,6 +1188,23 @@ std::string ErlUtil::formatBinary(unsigned char* buf, size_t size)
     return os.str();
 }
 
+std::string ErlUtil::formatAsString(char* buf, size_t size)
+{
+    return formatAsString((unsigned char*) buf, size);
+}
+
+std::string ErlUtil::formatAsString(unsigned char* buf, size_t size)
+{
+    std::ostringstream os;
+
+    for(unsigned iByte=0; iByte < size; iByte++)
```

Comment:
This one I'm going to leave as-is.  The important point is that ErlUtil::formatAsString() is called on erlang-formatted binaries that are not in general null-terminated (Things like <<"f2">>, which erlang will format as 2 bytes).  So passing the char buf directly to the string constructor can result in undefined behavior (garbage characters on the ends of otherwise-valid strings).  That's why this method copies the bytes into the ostringstream and explicitly null-terminates it before passing it to the string constructor.